examples/cpp-stream-client/main.cpp

```diff
@@ -19,8 +19,10 @@ class NoiseVideoSource : public VideoSource
     thread m_thread;
```

Comment:
This file was mostly just reformated. The two real changes are marked by comments